src/containers/PipelineResources/PipelineResources.js

```diff
@@ -105,12 +103,58 @@ export /* istanbul ignore next */ class PipelineResources extends Component {
       error,
       loading,
       namespace: selectedNamespace,
-      pipelineResources
+      pipelineResources,
+      intl
     } = this.props;
 
-    if (loading) {
-      return <StructuredListSkeleton border />;
-    }
+    const pipelineResourceActions = this.pipelineResourceActions();
+
+    const initialHeaders = [
+      {
+        key: 'name',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.name',
+          defaultMessage: 'Name'
+        })
+      },
+      {
+        key: 'namespace',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.namespace',
+          defaultMessage: 'Namespace'
+        })
+      },
+      {
+        key: 'type',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.type',
+          defaultMessage: 'Type'
+        })
+      },
+      {
+        key: 'dropdown',
```

Comment:
will this end up with the sort option displaying in the header on hover?

Carbon's DataTable provides support for an overflow menu in the last column: http://react.carbondesignsystem.com/?path=/story/datatable--with-overflow-menu

Reply:
I've had a look at the story and it looks like we will need to add a completely separate component to achieve this - could this be done under a separate issue?

Reply:
Looking at the story now it appears they just add an extra cell on each row containing the overflow (we can use our existing overflow menu as the content there), and leave out the dummy header. I'm not seeing anything that would require a new component. Can you point me to the piece that's concerning you? I'll try it out locally.

Reply:
Made the following changes locally and it works as expected (no sort option when hovering the overflow column header)

```diff
diff --git a/packages/components/src/components/Table/Table.js b/packages/components/src/components/Table/Table.js
index 599a948..40e1492 100644
--- a/packages/components/src/components/Table/Table.js
+++ b/packages/components/src/components/Table/Table.js
@@ -118,7 +118,7 @@ const Table = props => {
                       <TableSelectAll {...getSelectionProps()} />
                     )}
                     {headers.map(header => {
-                      return (
+                      return header.header ? (
                         <TableHeader
                           {...getHeaderProps({ header })}
                           className="cellText"
@@ -126,6 +126,8 @@ const Table = props => {
                         >
                           {header.header}
                         </TableHeader>
+                      ) : (
+                        <TableHeader />
                       );
                     })}
                   </TableRow>
```

```diff
diff --git a/src/containers/PipelineResources/PipelineResources.js b/src/containers/PipelineResources/PipelineResources.js
index eeef3dd..d181fc5 100644
--- a/src/containers/PipelineResources/PipelineResources.js
+++ b/src/containers/PipelineResources/PipelineResources.js
@@ -131,10 +131,7 @@ export /* istanbul ignore next */ class PipelineResources extends Component {
           defaultMessage: 'Type'
         })
       },
-      {
-        key: 'dropdown',
-        header: ''
-      }
+      { key: 'dropdown' }
     ];

     const pipelineResourcesFormatted = pipelineResources.map(
```